src/test/scala-2.11/markets/participants/MarketParticipantSpec.scala

```diff
@@ -90,12 +93,12 @@ class MarketParticipantSpec extends TestKit(ActorSystem("MarketParticipantSpec")
   feature("A MarketParticipant actor should be able to add and remove markets.") {
 
     val markets = mutable.Map.empty[Tradable, ActorRef]
-    val tickers = mutable.Map.empty[Tradable, Agent[immutable.Seq[Tick]]]
+    val tickers = mutable.Map.empty[Tradable, Agent[Tick]]
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2168327547/issues/source?bid=2921361&fileBranchId=3162043#l96)